examples/ConfigActuation/update_config.py

```diff
@@ -0,0 +1,124 @@
+# LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+# DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+# THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+# The views and conclusions contained in the software and documentation
+# are those of the authors and should not be interpreted as representing
+# official policies, either expressed or implied, of the FreeBSD
+# Project.
+#
+# This material was prepared as an account of work sponsored by an
+# agency of the United States Government.  Neither the United States
+# Government nor the United States Department of Energy, nor Battelle,
+# nor any of their employees, nor any jurisdiction or organization that
+# has cooperated in the development of these materials, makes any
+# warranty, express or implied, or assumes any legal liability or
+# responsibility for the accuracy, completeness, or usefulness or any
+# information, apparatus, product, software, or process disclosed, or
+# represents that its use would not infringe privately owned rights.
+#
+# Reference herein to any specific commercial product, process, or
+# service by trade name, trademark, manufacturer, or otherwise does not
+# necessarily constitute or imply its endorsement, recommendation, or
+# favoring by the United States Government or any agency thereof, or
+# Battelle Memorial Institute. The views and opinions of authors
+# expressed herein do not necessarily state or reflect those of the
+# United States Government or any agency thereof.
+#
+# PACIFIC NORTHWEST NATIONAL LABORATORY
+# operated by BATTELLE for the UNITED STATES DEPARTMENT OF ENERGY
+# under Contract DE-AC05-76RL01830
+
+# }}}
+
+import argparse
+import json
+
+from volttron.platform import get_address
+from volttron.platform.agent.known_identities import CONFIGURATION_STORE
+from volttron.platform.keystore import KeyStore, KnownHostsStore
+from volttron.platform.vip.agent.utils import build_agent
+
+def get_keys():
+    """Gets keys from keystore and known-hosts store"""
+    hosts = KnownHostsStore()
+    serverkey = hosts.serverkey(get_address())
+    key_store = KeyStore()
+    publickey = key_store.public
+    secretkey = key_store.secret
+    return {'publickey': publickey,
+            'secretkey': secretkey,
+            'serverkey': serverkey}
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('vip_id')
+    parser.add_argument('filename')
+    parser.add_argument('key')
+    parser.add_argument('value')
+    args = parser.parse_args()
+
+    vip_id = args.vip_id
+    filename = args.filename
+    key = args.key
+
+    try:
+        value = float(args.value)
+    except ValueError:
+        value = args.value
+
+
+    agent = build_agent(**get_keys())
+
+    files = agent.vip.rpc.call(CONFIGURATION_STORE,
```

Comment:
Consider using build_connection instead as then you won't need to specify the get(timeout) and you can pass the peer as a argument.  and you can just call "call"